pravega/pravega
# Issue 5969: Keep logs for 1 month by default

Signed-off-by: Igor Medvedev <medvedevigorek@gmail.com>

**Change log description**  
Fixes log retention policy to keep logs for 1 month

**Purpose of the change**  
Fixes #5969

**What the code does**  
Update `dist/conf/logback.xml` file to extend log history to 1 month.  

Updated only this logback configuration file as it's used in Pravega docker image by default

**How to verify it**  
Run docker image for couple of days and verify that kept log files cover for more than 30 hours
